brick/dockerlib.py

```diff
@@ -175,7 +177,7 @@ def get_image_names_with_dependency_hash(dependency_hash) -> List[str]:
         .strip()
     )
 
-    return images.split("\n")
+    return [s for s in images.split("\n") if s]
```

Comment:
Hmm, would it make more sense to use `filter()` for this? Maybe it's just me not being used to Python, it just seems like this could be more explicit and readable :)

Reply:
Fun fact: lambda, map() and filter() was close to being removed from Python 3... It backfired a bit, and they were kept there. I would argue that list compressions are often easier to read than lambda and filter. 

The filter alternative: `return filter(lambda s: s, images.split("\n"))`

https://www.artima.com/weblogs/viewpost.jsp?thread=98196

Reply:
Ah cool, didn't know that - in that case I think you're right :)